### DISCUSSION TITLE
AC7MAKER, how to?

### DISCUSSION THREAD

**Discussion by Dronard** [community/community#124091] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

@michgz

Hello, I would like to know how to use your program, I'm beginner on Python. I install Python and Pycharm, tried to start your program and I don't understand what I expect to see and what I have to do. My goal is create and add rythms on my Casio CT s1000v.

**Comment by ebndev** [comment, marked as answer]
Hi @Dronard,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from a specific project. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can but this is a public forum, so I will be closing this discussion.

Best of luck!

**Comment by ebndev** [comment]
Hi again @Dronard!

If you don't mind, would you mind letting us know how you initially ended up in the [Profile category](https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+) when creating your discussion and providing the link to the project's repository? We are trying to investigate if there is a bug or link that may be re-directing users from the project's Discussions board, if that’s where you intended to post.

Any insight would be helpful, thanks!

**Reply by ebndev** [reply]
Never mind, found the issue! Thanks again for your time and being a part of the GitHub Community 😄
